core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/Progress.java

```diff
@@ -0,0 +1,39 @@
+package uk.ac.bbsrc.tgac.miso.core.data.workflow;
+
+import static uk.ac.bbsrc.tgac.miso.core.data.workflow.Workflow.WorkflowType;
+
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Set'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)